examples/secureTokenAndroidKeyStore/src/main/java/io/realm/examples/securetokenandroidkeystore/MainActivity.java

```diff
@@ -86,29 +83,21 @@ protected void onResume() {
     // build SyncConfiguration with a user store to store encrypted Token.
     private void buildSyncConf() {
         // the rest of Sync logic ...
-        SyncUser user = createTestUser(Long.MAX_VALUE);
-        String url = "realm://objectserver.realm.io/default";
-        SyncConfiguration secureConfig = new SyncConfiguration.Builder(user, url).build();
-        Realm realm = Realm.getInstance(secureConfig);
-        // ...
-    }
-
-    // Helpers
-    private final static String USER_TOKEN = UUID.randomUUID().toString();
+        SyncCredentials credentials = SyncCredentials.usernamePassword("username", "password");
+        final String urlAuth = "http://objectserver.realm.io:9080/auth";
+        final String url = "realm://objectserver.realm.io/default";
+
+        SyncUser.loginAsync(credentials, urlAuth, new SyncUser.Callback<SyncUser>() {
+            @Override
+            public void onSuccess(SyncUser user) {
+                SyncConfiguration secureConfig = new SyncConfiguration.Builder(user, url).build();
+                Realm realm = Realm.getInstance(secureConfig);
+                // ...
```

Comment:
close realm after this?

Reply:
We can, although this is just an example, the emphasis is on the SecureStore ...

Reply:
yes, it is like that before. So it is fine :)